domains.json

```diff
@@ -68,6 +71,11 @@
     "1inch.io",
     "mantle.xyz",
     "friend.tech",
+    "balancer.fi",
+    "hiddenhand.finance",
+    "garden.finance",
```

Comment:
not sure what this is, seems too small to be here

Reply:
This is from the original REN team - BTC atomic swaps. Its on defillama, but fair enough to leave it out.